From 89b2696e14aedcf6dc9c4f3da64bdf88be98a8c7 Mon Sep 17 00:00:00 2001
From: Marko Kocic <markokocic2000@gmail.com>
Date: Mon, 24 Feb 2025 20:13:39 +0100
Subject: [PATCH 2/2] Add path rewrites for set/reset password

---
 next.config.ts | 29 +++++++++++++++++++++++++++++
 1 file changed, 29 insertions(+)

diff --git a/next.config.ts b/next.config.ts
index 5e891cf..c0bf3f0 100644
--- a/next.config.ts
+++ b/next.config.ts
@@ -4,4 +4,33 @@ const nextConfig: NextConfig = {
   /* config options here */
 };
 
+module.exports = {
+    async rewrites() {
+        return [
+            {
+                source: '/i/s',
+                has: [
+                    {
+                        type: 'query',
+                        key: 't',
+                        value: '(?<t>.*)'
+                    }
+                ],
+                destination: '/auth/password/set/:t'
+            },
+            {
+                source: '/i/r',
+                has: [
+                    {
+                        type: 'query',
+                        key: 't',
+                        value: '(?<t>.*)'
+                    }
+                ],
+                destination: '/auth/password/reset/:t'
+            }
+        ]
+    },
+}
+
 export default nextConfig;
-- 
2.48.1

